### PR TITLE
Update extended const in features.json

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: features.json [hunk 1]
```diff
@@ -179,6 +179,7 @@
 				"bigInt": true,
 				"bulkMemory": true,
 				"exceptions": ["flag", "Requires flag `--experimental-wasm-eh`"],
+				"extendedConstantExprs": ["flag", "Requires flag `--experimental-wasm-extended-const`"],
```
Comment (dschuff) [inline features.json:182]: This is a JS flag, right? Not a Chrome flag? I think it needs to be wrapped in `--js-flags=`

Comment (dschuff) [reply features.json:182]: (Also we should check whether it's enabled by the `#enable-experimental-webassembly-features` flag in chrome). That would be more convenient than requiring actual command-line switches.

Comment (sbc100) [reply features.json:183]: This is the Node section.. so that `--js-flags` things is not needed.

File: features.json [hunk 2]
```diff
@@ -94,6 +94,7 @@
 				"bigInt": "85",
 				"bulkMemory": "75",
 				"exceptions": "95",
+				"extendedConstantExprs": ["flag", "Requires flag `--js-flags=--experimental-wasm-extended-const`"],
```
Comment (sbc100) [inline features.json:97]: Confirmed that `#enable-experimental-webassembly-features` does not work here.

**Review by dschuff — APPROVED** [review]

**Comment by sbc100** [comment]
Does this website auto-publish somehow?

**Comment by dschuff** [comment]
Yeah it seems to be updated already.

**Pull request merged**